modules/tide_cms_support/tide_cms_support.api.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/**
+ * @file
+ * Hooks for the Tide CMS Support system.
+ */
+
+/**
+ * @addtogroup hooks
+ * @{
+ */
+
+/**
+ * Provide online user help.
+ *
+ * By implementing hook_tide_help(), a module can make documentation available
+ * to CMS users for specific pages.
+ *
+ * The page-specific help information provided by this hook appears in the
+ * Tide CMS Help block (provided by the core Help module), if the block is
+ * displayed on that page.
+ *
+ * For detailed usage examples of:
+ * - Page-specific help using only routes, see book_help().
+ * - Page-specific help using routes and $request, see block_help().
+ *
+ * @param string $route_name
+ *   For page-specific help, use the route name as identified in the
+ *   module's routing.yml file.
+ * @param \Drupal\Core\Routing\RouteMatchInterface $route_match
+ *   The current route match. This can be used to generate different help
+ *   output for different pages that share the same route.
+ *
+ * @return string|array
+ *   A render array, localized string, or object that can be rendered into
+ *   a string, containing the help text.
+ */
+function hook_tide_help(string $route_name, \Drupal\Core\Routing\RouteMatchInterface $route_match) {
```

Comment:
hey @christopher-hopper, It would be good if importing with the `use` operator.

Reply:
@vincent-gao 

Good pickup, but... 

The `use` operator isn't included into Drupal `.api.php` files so that developers implementing the hook are able to copy the full qualified classes into their hook implementations. It seems like you would add `use` going on other coding standards, but the `.api.php` file is a special case. These files are more-or-less documentation files. 

For an example in core, take a look at 

- `docroot/core/core.api.php`
- `docroot/core/modules/help/help.api.php`

and see the API documentation at 

- https://api.drupal.org/api/drupal/core%21core.api.php/group/hooks/9.4.x 
- https://api.drupal.org/api/drupal/core%21core.api.php/9.4.x